### ISSUE TITLE
[Question] [Galleria] aria-label

### ISSUE THREAD

**Issue by linuxissuper** [primefaces/primeng#19080]
The docs say: 
>Galleria uses region role and since any attribute is passed to the main container element, attributes such as aria-label and aria-roledescription can be used as well

Does this mean, `aria-label` doesn't have to be on the element with `region="role"`? 
If so, where in the spec can I find this?

Right now I'm using the `IBM Equal Access Accessibility Checker` browser extension and I'm getting this error: `Element with "region" role does not have a label`

<img width="1931" height="82" alt="Image" src="https://github.com/user-attachments/assets/c4034e5b-5e4d-4c62-a5d8-33e260d9b5d0" />
